main/src/main/java/org/mobilitydata/gtfsvalidator/runner/ValidationRunner.java

```diff
@@ -100,7 +101,10 @@ public Status run(ValidationRunnerConfig config) {
     GtfsFeedContainer feedContainer;
     GtfsInput gtfsInput = null;
     try {
-      gtfsInput = createGtfsInput(config, versionInfo.currentVersion().get(), noticeContainer);
+      Optional<String> optionalValue = versionInfo.currentVersion();
+      if (optionalValue.isPresent()) {
+        gtfsInput = createGtfsInput(config, versionInfo.currentVersion().get(), noticeContainer);
```

Comment:
You can use `optionalValue ` instead of `versionInfo.currentVersion()` on this line

Reply:
If this changes the scope of #1640. If not, I suggest adding it as its own PR.

Reply:
I don't know why it needs its own PR. 
It's simply replacing `versionInfo.currentVersion().get()` with `optionalValue .get()` since we already have optionalValue.
We could also keep it as is it would not be a big deal.

Reply:
It has no relation to the scope of the issue. Any bug/side effect on this line will require reverting the commit.

Reply:
In this case, it introduces an issue. If the version is not resolved, _gtfsInput_ will be null. The primary process of the validator should not be blocked by the version resolution. The subsequence code consuming the version should support a scenario with no version. I suggest passing the _optionalValue_ as a parameter or using _Optional.orElse(null)_ to pass null avoiding the NoSuchElement exception from _Optional.get_.

Reply:
Thanks for the clarification. My point was that the central issue of these few lines is fixing. Currently(master), if the version is not resolved _Optional. get_ throws an exception. The protection _Optional.isPresent_ it's avoiding the exception but skipping validation that results in _a similar_ end-user experience. I suggest ensuring the feed is processed even if the version is not resolved.

Reply:
This change is reverted coz it's not needed after the agency.txt is replaced by networks.txt in GtfsNetworkSchema